base/src/org/compiere/model/MInOut.java

```diff
@@ -1111,10 +1111,15 @@ public String prepareIt()
 		//	Credit Check
 		if (isSOTrx() && !isReversal())
 		{
-			I_C_Order order = getC_Order();
+			MOrder order = (MOrder)getC_Order();
+			Boolean IsApprovedForDelivery = false;
+			if (order !=null)
+				IsApprovedForDelivery = order.get_ValueAsBoolean("IsApprovedFordelivery");
```

Comment:
Please add **IsApprovedFordelivery** to **C_Order** table for complete test

Reply:
Hi @yamelsenih ,
I added the missing XML.